styles/styles.css

```diff
@@ -1,10 +1,8 @@
 * {
```

Comment:
- [Optional] Consider making your app responsive over all screen sizes. This is so that users can access it using various devices.